flink-cdc-common/src/main/java/com/ververica/cdc/common/factories/Factory.java

```diff
@@ -19,7 +19,10 @@
 import org.apache.flink.configuration.ConfigOption;
```

Comment:
Can you also update this import to com.ververica.cdc.common.configuration.ConfigOption?

Reply:
I will fix this in another PR.